### ISSUE TITLE
Review CRC cards

### ISSUE THREAD

**Issue by kezzayuno** [TaskToLife/TaskToLife#14]
- Discuss at Python meeting 
- Increase cohesion and coupling 
- Make sure it is in sync with User Stories and UML

**Comment by Holy-Hero** [comment]
I see no big issues, but we may need to think of a way to reduce the amount of stuff that Task and Player have

**Comment by kezzayuno** [comment]
Good point, I thought it was an issue too. Feel free to edit it during the meeting to how you think it'll work backend. Topic for the Python and Leader meeting I think
